docs/glossary.md

```diff
@@ -0,0 +1,468 @@
+
+  You'll find this directory at the root of your Top or in the `result` directory if using EPNix.
+
+  The binary executable file(s) will end up there.
+  This binary is call from the [Shebang](https://en.wikipedia.org/wiki/Shebang_(Unix))
+  at the very start of the `.cmd` file,
+  allowing to "run" your IOC.
+
+**`lib` directory**
+  This directory is created into your Top after building it,
+  either with `make` or with `nix build` if using EPNix.
+
+  You'll find this directory at the root of your Top or in the `result` directory if using EPNix.
+
+  The library file(s) will end up there.
+
+**`db` directory**
+  This directory is created into your Top after building it,
+  either with `make` or with `nix build` if using EPNix.
+
+  You'll find this directory at the root of your Top or in the `result` directory if using EPNix.
+
+  All the `.db` files will end up being copied there,
+  allowing to find them with more ease
+  (e.g. at runtime from the `.cmd` file, or from the IOC Shell).
+
+**`dbd` directory**
+
+  This directory is created into your Top after building it,
+  either with `make` or with `nix build` if using EPNix.
+
+  You'll find this directory at the root of your Top or in the `result` directory if using EPNix.
+
+  All the `.dbd` files will end up being copied there,
+  allowing to find them with more ease
+  (e.g. at runtime from the `.cmd` file, or from the IOC Shell).
+
+**`configure` directory**
+  This directory is created into your Top after running the `makeBaseApp.pl` EPICS command.
+
+  You'll find this directory at the root of your Top (and in the `result` directory if using EPNix).
+
+  It might contains some of the bellow configuration files:
+
+  - `CONFIG`: EPICS configuration file for builds;
+  - `CONFIG_SITE`: EPICS configuration file for application-specific builds;
+  - `CONFIG_SITE.local`: EPICS configuration file allowing to override CONFIG_SITE without having to modify it;
+  - `RELEASE`: EPICS configuration file for base and external support modules location;
+  - `RELEASE.local`: EPICS configuration file allowing to override RELEASE without having to modify it;
+  - `RULES`: EPICS configuration file including the appropriate rules configuration file;
+  - `RULES.ioc`: EPICS build configuration file of the iocBoot/ sub-directorie(s);
+  - `RULES_DIRS`: EPICS build configuration file of each subdirectory;
+  - `RULES_TOP`: EPICS configuration file specific to a Top.
+
+  :::{seealso}
+
+  - <https://docs.epics-controls.org/en/latest/build-system/specifications.html#configuration-files>
+
+  :::
+
+**CA**
+  Channel Access (CA) is the default communication protocol
+  used between EPICS servers (i.e. IOCs)
+  and EPICS clients
+  (i.e. HMI monitoring tools, archiving softwares, alarms systems, etc).
+  It is mostly used to share PVs information across a network.
+  CA will use UDP to initiate a client-server communication,
+  and then use TCP for the rest of the communication.
```

Comment:
I think the UDP connection depends on `EPICS_CA_AUTO_ADDR_LIST` and `EPICS_CA_ADDR_LIST`. I think it's best to leave the technical part of the protocol to other articles and resources, like the ones you've linked.


```suggestion
```